src/dos/drive_iso.cpp

```diff
@@ -192,7 +192,7 @@ int isoDrive::UpdateMscdex(char drive_letter, const char *path, uint8_t &sub_uni
 		CDROM_Interface *cdrom = new CDROM_Interface_Image(sub_unit);
```

Comment:
It's a C style array storing raw pointers.  It could probably be refactored but I'm going to say out of scope for this PR (because, again, I barely even looked at this file).